keadm/cmd/keadm/app/cmd/edge/join.go

```diff
@@ -253,6 +253,9 @@ func createEdgeConfigFiles(opt *common.JoinOptions) error {
 	if v.Major <= 1 && v.Minor < 12 {
 		return createV1alpha1EdgeConfigFiles(opt)
 	}
+	if v.Major >= 1 && v.Minor >= 14 && opt.RuntimeType != constants.DefaultRuntimeType {
+		return fmt.Errorf("since KubeEdge v1.14, runtime type only supports `remote`")
```

Comment:
If users want to use Docker, the runtime type still set to `remote`, then edged connect to `cri-dockerd`?

Reply:
> If users want to use Docker, the runtime type still set to `remote`, then edged connect to `cri-dockerd`?

set runtimeType to remote and set `remoteRuntimeEndpoint` to `var/run/cri-dockerd.sock`, edged will connect to `cri-dockerd`.  See more detail in #4843

Reply:
Okey :)